### PR TITLE
Initiate history page - feature-v5

### LINKED ISSUE #191: Initiate History page
**Is your feature request related to a problem? Please describe.**
- Currently, the history page hasn't been initiated.

**Describe the solution you'd like**
- Initiate the history page using Jekyll.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

- The purpose of this PR is to fix #191 

## Goals

- Implement the history page using Jekyll.

## Approach

- Create a separate directory for the history page.
- Create a new layout for the history page and history-content page.
- Load the .md files in the _posts folder to the history page

### Screenshots

  https://user-images.githubusercontent.com/27630091/116001797-0944e980-a614-11eb-8f80-aae94116cee9.mov

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: _posts/2020-04-22-2002.markdown [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+layout: post
```
Comment (jayasanka-sack) [inline _posts/2020-04-22-2002.markdown:2]: Shall we change the layout names? (Ex: history_page)

File: _posts/2020-04-22-2002.markdown [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <li>Vice Captain: Robert H. Anderson</li>
+    <li>Drake Austin</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Badminton</summary>
+  <ul>
+    <li>Captain: John Smith</li>
+    <li>Vice Captain: Robert H. Anderson</li>
```
Comment (jayasanka-sack) [inline _posts/2020-04-22-2002.markdown:54]: Is there a way to avoid having HTML, but only the data?

File: history/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                </div>
+              </div>
+            </div>
+          </a>
+          {% endfor%}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (jayasanka-sack) [inline history/index.html:39]: Let's add a new line

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks @kumuditha-udayanga !

i've added some review comments.

btw, shall we try to get rid of html in data pages?

the goal is to have a mechanism that anyone can update the data file easily without having an html knowledge.


Also the filenames of markdown files should be changed to the year. Ex: 2012.markdown.  (We can use .md instead of .markdown , btw)

**Comment by kumuditha-udayanga** [comment]
@jayasanka-sack, I've update the PR :)

**Review by jayasanka-sack — APPROVED** [review]
Thanks @kumuditha-udayanga !

**Pull request merged**